### PR TITLE
flagmaster info.md PR

### PR DESCRIPTION
**Pull request by RayDeeUx**

### CODE REVIEW

**Review by j10a1n15 — DISMISSED** [review]
I agree

File: info.md [hunk 1]
```diff
@@ -167,4 +167,56 @@ Example:
...
+_ The nested ``"currencysymbols"`` object (containing the IRL cost of the 675 gems bundle in each currency before discounts, taxes, or othern fees)
+- The nested ``"currencysymbols"`` object (containing the [Minecraft-supported] currency symbol as strings)
+- The nested ``"currencysymbolsprecedes"`` object (containing whether the currency symbol should be placed before or after the monetary value as booleans)
+
+An abridged example of the object is provided below.
+
+Example:
+```json
+{
+  "ingame": {
```
Comment (Su386yt) [inline info.md:206]: Could you add what the ingame and storehypixelnet objects are?

**Review by Su386yt — CHANGES_REQUESTED** [review]

File: info.md (hunk 1 above)
Comment (Su386yt) [reply info.md:206]: Sorry, thats the biggest reason I couldn't do it myself. I had no idea what these were

Comment (j10a1n15) [reply info.md:206]: Su dont agree

Comment (Su386yt) [reply info.md:206]: immm sorryyyyy

Comment (Su386yt) [reply info.md:206]: bump

**Comment by Su386yt** [comment]
Bump

**Comment by Su386yt** [comment]
im gonna close this soon lol for inactivity

**Comment by j10a1n15** [comment]
Is this fine now, except for the conflicts obv

**Review by Su386yt — APPROVED** [review]

**Pull request merged**